tools/ms2snoop/.conda-lock.yml

```diff
@@ -0,0 +1,1383 @@
+# This lock file was generated by conda-lock (https://github.com/conda-incubator/conda-lock). DO NOT EDIT!
```

Comment:
this file should not be here, isn't it?

Reply:
Thanks, I will remove it!

Reply:
What do you think about the environment.yml file..?

And also, for reproducible deploy (and not have to resolve a tool's dependencies) - how should I provide the conda lock file?

Thanks!

Reply:
environment.yml files seem to be correct, but should also not end up in the TS I guess. For really 99% reproducible deploys we use containers. So there is one container created for your tool already that has all your dependencies. A frozen container for your environment.yml file.

Reply:
I understand, thanks!
I'll fix that.